CHANGELOG.md

```diff
@@ -4,11 +4,34 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Master
+
+### Added
+
+- Added concurrency backend autodetection.
```

Comment:
So, e.g…

```suggestion
- Added concurrency backend autodetection. (Pull #585)
```

(That's why we tend to edit the title of the PR squash commit — personally, I just go to `/compare` and copy/paste the entries here, making small copy edits if needed.)

Reply:
Yes great call.
Will start to make sure that titles get reviewed at the point of merge.
I've erronously got two cases where I've had to reference a commit rather than a PR.

We'll want to get really tight about *only ever* allowing changes-as-PRs, partly because it's *so much nicer* for history/interlinking, and partly to make sure *everything* is getting fully reviewed.